ICP-2082: Set initial state of Z-Wave Switch Generic

munds opened the pull request.

skt123 committed:
[DVCSMP-3206]
Adding Fix for Aeon Labs Panic Button and Aeon Key Fob

Commit:
[DHF-58] Exponentially pings offline Z-Wave Water Valve

Commit:
[DHF-59] Exponentially pings offline Z-Wave Garage Door Opener

workingmonk committed:
Merge pull request #2442 from SmartThingsCommunity/master

Rolling up master to staging

workingmonk committed:
Merge pull request #2463 from SmartThingsCommunity/master

Rolling up master to staging

munds committed:
Set initial state of Z-Wave Switch Generic

munds committed:
Set initial state of Z-Wave Switch Generic

workingmonk committed:
Merge pull request #2306 from skt123/aeon_panic

[DVCSMP-3206] Adding Fix for Aeon Labs Panic Button

workingmonk committed:
Merge pull request #2416 from jackchi/dhf-58

[DHF-58] Exponentially pings offline Z-Wave Water Valve

workingmonk committed:
Merge pull request #2417 from jackchi/dhf-59

[DHF-59] Exponentially pings offline Z-Wave Garage Door Opener

dkirker committed:
PROB-1673 Don't use defaultValue and required in the preferences block

vaish-rahul committed:
[DVCSMP-3355] [DVCSMP-3352] [DVCSMP-3353] [DVCSMP-3356] Kwikset Z-wave
- Description text is displayed as "Locked by Master code" when lock
is locked using 'Lock' button.
Yale Z-wave - Incorrect state is shown for Unlock event in DTH when
Autolock is enabled.
Yale Zigbee lock- State is not changed in the DTH after locking via
app when Auto Lock is ON.
Change "Locked/Unlocked via app" to "Locked/Unlocked".

workingmonk committed:
Merge pull request #2465 from vaish-rahul/staging

[DVCSMP-3355] [DVCSMP-3352] [DVCSMP-3353] Kwikset Z-wave - Description

vaish-rahul committed:
[ICP-2051] Added new fingerprint for YRD221 models of older generation
in Chile. Without the fingerprint the models of older generation will
not be onboarded with device type as Zigbee Lock.

marstorp committed:
PROB-1869 Zen thermostat connecting with no battery state (#2424)

* PROB-1869 Zen thermostat connecting with no battery state
Changing max allowed battery voltage from 6.0 to 6.1 as
logs from a user showed the thermostat reporting 6.1V despite
there's only 4 x 1.5V batteries.
Also made a change so that thermostatMode and fanMode only generate
an event when the mode is changing.

* Update zen-thermostat.groovy

dkirker committed:
Merge pull request #2468 from dkirker/PROB-1673

PROB-1673 Don't use defaultValue and required in the preferences block

workingmonk committed:
Merge pull request #2472 from SmartThingsCommunity/staging

Rolling down staging to master

workingmonk committed:
Merge pull request #2473 from SmartThingsCommunity/master

Rolling up master to staging

GeorgeHahn committed:
Remove unused polling capabilities

workingmonk committed:
Merge pull request #2470 from vaish-rahul/staging

[ICP-2051] Added new fingerprint for YRD221 models of older generation

GeorgeHahn committed:
Merge pull request #2476 from SmartThingsCommunity/bugfix/remove-unused-polling

[DVCSMP-3370] Remove unused polling capabilities

vaish-rahul committed:
[DVCSMP-3377] Schlage Locks - For some locks DTH is assigning default
code name instead of code name provided by user.

workingmonk committed:
Merge pull request #2490 from vaish-rahul/staging

[DVCSMP-3377] Schlage Locks - For some locks DTH is assigning default

workingmonk committed:
Merge pull request #2496 from SmartThingsCommunity/staging

Rolling down staging to master

workingmonk committed:
Merge pull request #2497 from SmartThingsCommunity/master

Rolling up master to staging

workingmonk committed:
DVCSMP-3283 Adding blacklisted DTH to catch bad devices (#2371)

* DVCSMP-3283 Adding blacklisted DTH to catch bad devices

* Adding correct support link

newtectonics committed:
Added curb bridge to smartthingspublic (#2346)

* Added curb bridge to smartthingspublic

* untabified, appsettings used

* switched to 1 minute interval

* took out historical data

* took out a rouge }

* Added Energy Monitor with load controller and thermostat controls

* added historical again, added refresh tokens, added better state restore logic

* switches intelligently generate default values

* syntax error

* typo

* it works!

* fixed the error messages popping up in the live logging window

* added 1m resolution, got rid of erraneous error messages

* took out unnecessary debug messages

* added all the different energy timeintervals

* finishing touches on the curb app

* Took out a debug statement

* reduced api endpoints to only have single power/energy values

* tabs to spaces

* fixed atomicstate looping

workingmonk committed:
DVCSMP-3384 Add tile type to multi attribute tile for devices

workingmonk committed:
Merge pull request #2501 from workingmonk/bug/multi_tiles

DVCSMP-3384 Add tile type to multi attribute tile for devices

GeorgeHahn committed:
Add the runLocally flag to Z-Wave Generic DTHs

marstorp committed:
DVCSMP-3374 Zen Thermostat has no settings for "Heat only" or "Cool only" (#2503)

A customer reported on PR 2296;
"My Zen can only Heat (10A setting I think). There is no option for "Off, Heat" in the thermostat modes settings UI."
Adding ["off", "heat"] and ["off", "cool"] to settings.

workingmonk committed:
Merge pull request #2505 from SmartThingsCommunity/staging

Rolling down staging to master

juano2310 committed:
DVCSMP-3283 - Blacklisted Device Link to Support

fingerprints

install

workingmonk committed:
Merge pull request #2506 from juano2310/blacklist

DVCSMP-3283 - Blacklisted Device Link to Support

workingmonk committed:
Merge pull request #2504 from GeorgeHahn/CHAD-1596

[CHAD-1596] Add the runLocally flag to Z-Wave Generic DTHs

greens committed:
DVCSMP-690 (#2483)

* DVCSMP-690 Aeon multisensor improvements

* Removes the configure tile.
* Configuration commands should now be sent on wake-up, until a temperature report is sent
* Added preferences to configure the motion timeout and the reporting interval. Defaults are 20s and 5min
* Send an association remove during configuration to prevent double reports
* Removes the battery from the sensor reporting group. It now reports every 12 hours
* Renamed/reworked preferences to more closely match aeon-multisensor-6
* Configuration will now happen on preference update or wakeup if it hasn't happened yet
* Added some extra description text to the configuration preferences.

vaish-rahul committed:
[ICP-2233] During scanning of Lock Codes, Recently tab shows multiple
events for added lock codes.

mckeed commented on devicetypes/smartthings/zwave-switch-generic.src/zwave-switch-generic.groovy:
```diff
@@ -64,11 +64,13 @@ metadata {
 def updated(){
 // Device-Watch simply pings if no device events received for checkInterval duration of 32min = 2 * 15min + 2min lag time
 	sendEvent(name: "checkInterval", value: 2 * 15 * 60 + 2 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
+	response(refresh())
```
I think it's only needed in installed(), not updated() too

workingmonk committed:
Merge pull request #2516 from SmartThingsCommunity/staging

Rolling down staging to master

workingmonk committed:
Merge pull request #2517 from SmartThingsCommunity/master

Rolling up master to staging

greens committed:
DVCSMP-2220 Aeon Siren not sending alert on secure inclusion failure (#2511)

This just cribs the way other secure devices handle this failure. In the future we'd like to have this standardized or done automatically.

workingmonk committed:
Merge pull request #2512 from vaish-rahul/staging

[ICP-2233] During scanning of Lock Codes, Recently tab shows multiple

workingmonk committed:
Merge pull request #2518 from SmartThingsCommunity/staging

Rolling down staging to master

munds committed:
Merge branch 'ICP-2082' of github.com:munds/SmartThingsPublic into ICP-2082

munds committed:
Call response(refresh()) only in installed()

munds commented:
@mckeed I've made the changes you requested 👍 

Can you please do a code review whenever you get a chance!

cc: @workingmonk

munds commented:
Whoa hold on, this doesn't look right. Let me redo the PR. :)

munds commented:
New PR is https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/2522
